src/backend/utils/load/ag_load_edges.c

```diff
@@ -171,8 +174,10 @@ int create_edges_from_csv_file(char *file_path,
     cr.object_id = object_id;
 
     while ((bytes_read=fread(buf, 1, 1024, fp)) > 0) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{